Advanced_Projects/Resume Scanner/resume_scanner.py

```
@@ -0,0 +1,43 @@
+def extract_phone(text):
+    match = re.search(r"(\+?\d{1,3}[\s-]?)?(\(?\d{3}\)?[\s-]?)?\d{3}[\s-]?\d{4}", text)
+    return match.group(0) if match else None
+
+def extract_name(text):
+    doc = nlp(text)
+    for ent in doc.ents:
+        if ent.label_ == "PERSON":
+            return ent.text
+    return None
+
+def extract_skills(text):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)